src/Client/OAuth2ClientInterface.php

```diff
@@ -39,7 +39,7 @@ public function redirect(array $scopes, array $options);
      * @throws \KnpU\OAuth2ClientBundle\Exception\MissingAuthorizationCodeException
      * @throws \League\OAuth2\Client\Provider\Exception\IdentityProviderException   If token cannot be fetched
      */
-    public function getAccessToken();
+    public function getAccessToken(array $options = []);
```

Comment:
We should describe this an `@param` - something like:

> Additional options that should be passed to the getAccessToken() of the underlying provider

Reply:
Ready :wink: